From 039bf2a81797b8f3af6058d34005a4896a363221 Mon Sep 17 00:00:00 2001
From: Darren Tucker <dtucker@dtucker.net>
Date: Fri, 7 Sep 2018 14:06:57 +1000
Subject: [PATCH 003/294] Initial len for the fmt=NULL case.

Patch from jjelen at redhat via bz#2687.  (OpenSSH never calls
setproctitle with a null format so len is always initialized).
---
 openbsd-compat/setproctitle.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/openbsd-compat/setproctitle.c b/openbsd-compat/setproctitle.c
index 2b15c6e..dbd1a95 100644
--- a/openbsd-compat/setproctitle.c
+++ b/openbsd-compat/setproctitle.c
@@ -125,7 +125,7 @@ setproctitle(const char *fmt, ...)
 #if SPT_TYPE != SPT_NONE
 	va_list ap;
 	char buf[1024], ptitle[1024];
-	size_t len;
+	size_t len = 0;
 	int r;
 	extern char *__progname;
 #if SPT_TYPE == SPT_PSTAT
-- 
1.8.3.1

